woofbot-lnd/hooks/pre-start

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+APP_DATA_DIR="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/..)/data"
+
+[ ! -d "${APP_DATA_DIR}" ] && mkdir -p "${APP_DATA_DIR}" && chown 1000:1000 "${APP_DATA_DIR}"
+
+SERVER_LOGS_DIR="${APP_DATA_DIR}/server-logs"
+
+[ ! -d "${SERVER_LOGS_DIR}" ] && mkdir -p "${SERVER_LOGS_DIR}" && chown 1000:1000 "${SERVER_LOGS_DIR}"
```

Comment:
Is this pre-start hook needed? Umbrel already exports an `APP_DATA_DIR` env var automatically for your app, and you have .gitkeep folders under the required directories.

Reply:
Ah, I was just browsing GitHub Issues for this repo and came accross https://github.com/getumbrel/umbrel-apps/issues/346. I see what you are doing here now.

Reply:
Yes, it's the same code from the original WoofBot. It's needed for the MongoDB database.

Reply:
This pre-start hook is not needed because this is a new app and therefore the `server-logs` directory will exist from day 1 and will have the correct ownership. In the case of #346, the hook was needed to address the case of an app update (not app submission) not creating the directory.

Reply:
@nevets963 Removed the hook.